ros_ign_interfaces/srv/DeleteEntity.srv

```diff
@@ -0,0 +1,5 @@
+string name                       # Name of the Ignition Gazebo entity to be deleted. 
+uint8 type                        # Type: Model or Light
```

Comment:
How about adding an entity message like [ignition::msgs::Entity](https://github.com/ignitionrobotics/ign-msgs/blob/ign-msgs7/proto/ignition/msgs/entity.proto) and using it here? Using the same pattern as Ignition should make it easier to maintain. It's also easier to only update one messsage if we add new entity types.